txnkv/txnlock/lock_resolver.go

```diff
@@ -350,11 +356,12 @@ func (lr *LockResolver) BatchResolveLocks(bo *retry.Backoffer, locks []*Lock, lo
 
 // ResolveLocksOptions is the options struct for calling resolving lock.
 type ResolveLocksOptions struct {
-	CallerStartTS uint64
-	Locks         []*Lock
-	Lite          bool
-	ForRead       bool
-	Detail        *util.ResolveLockDetail
+	CallerStartTS   uint64
+	Locks           []*Lock
+	Lite            bool
+	ForRead         bool
+	Detail          *util.ResolveLockDetail
+	PessimisticLite bool
```

Comment:
I'm thinking about: do we realy need `PessimisticLite`? can we just use `Lite` for both optimistic and pessimistic.

Reply:
Yes, I'am thinking about risks about using region resolve for all the expired pessimistic locks, if there is some situations cluster stability would be afffected.